source/setup.rst

```diff
@@ -100,7 +100,7 @@ The installation is possible by adding our own repository
 
 .. code-block:: sh
 
-    composer create-project phpmyadmin/phpmyadmin --repository-url=https://www.phpmyadmin.net/packages.json
+    composer create-project phpmyadmin/phpmyadmin --repository=https://www.phpmyadmin.net
```

Comment:
No it's not. That has been deprecated as --repository handles URLs *and* complete repo definitions.